src/main/java/ua/lviv/iot/parkingServer/datastorage/AbstractFIleStore.kt

```diff
@@ -0,0 +1,46 @@
+package ua.lviv.iot.parkingServer.datastorage
+
+import ua.lviv.iot.parkingServer.model.Record
+import ua.lviv.iot.parkingServer.utils.Util
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStreamWriter
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.*
+
+abstract class AbstractFIleStore<T : Record> {
+    fun readRecords(): List<T> {
+        val fileName = "$RESULT_FOLDER/${getRecordName()}-${Util.getDateTodayInString()}"
+        return if (Files.exists(Paths.get(fileName))) readRecordsFrom(File(fileName)) else LinkedList()
```

Comment:
Just use `emptyList()` instead of `LinkedList()`

Reply:
✅